go.mod

```diff
@@ -41,11 +41,12 @@ require (
 	github.com/evanphx/json-patch v4.11.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-bindata/go-bindata v3.1.1+incompatible
+	github.com/go-openapi/spec v0.20.3
```

Comment:
what is adding this back in? we've worked hard to drop it, and are in the process of adding a presubmit to keep it from being reintroduced

Reply:
Added the full sequence in the first comment/ PR description.